src/10_functions.py

```diff
@@ -1,7 +1,13 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(num):
+    if(int(num) % 2 == 0):
+        return True
+    else:
+         return False
```

Comment:
a tip I learned at a code challenge for an interview... you can shortcut this by just returning the condition...
```
def is_even(n):
    return n % 2 == 0
```